dffml/operation/model.py

```diff
@@ -35,6 +35,62 @@ def __post_init__(self):
     ctx_enter={"mctx": (lambda self: self.parent.model())},
 )
 async def model_predict(self, features: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Predict using dffml models.
+
+    Parameters
+    ++++++++++
+    features : dict
+        A dictionary contaning feature name and feature value.
+
+    Returns
+    +++++++
+    dict
+        A dictionary containing prediction.
+
+    Examples
+    ++++++++
+
+    The following example shows how to use model_predict.
+
+    >>> slr_model = SLRModel(
+    ... features=Features(DefFeature("Years", int, 1)),
```

Comment:
Can you take this, put it in a python file, run black on it, and put it back here please? Thanks!

Reply:
Ah! Thanks, you caught this, I did it for IO examples and missed this one. My bad!